clients/drcachesim/analyzer.cpp

```diff
@@ -339,6 +339,16 @@ analyzer_tmpl_t<RecordType, ReaderType>::init_scheduler_common(
             uint64_t filetype = scheduler_.get_stream(i)->get_filetype();
             VPRINT(this, 2, "Worker %d filetype %" PRIx64 "\n", i, filetype);
             if (TESTANY(OFFLINE_FILE_TYPE_CORE_SHARDED, filetype)) {
+                if (i == 0 && shard_type_ == SHARD_BY_CORE) {
+                    // This is almost certainly user error.
+                    // Better to exit than risk user confusion.
+                    // Ideally this could be detected in analyzer_multi but that is
```

Comment:
File an issue on potentially simplifying it later? Also include why is it better to detect in analyzer instead of scheduler?

Reply:
Filed #7045 and referenced here.